app/initializers/ember-cli-mirage.js

```diff
@@ -22,11 +23,13 @@ export default {
         server.loadConfig(testConfig);
       }
 
-      if (env === 'test' && factoryMap) {
-        server.loadFactories(factoryMap);
-
+      if (env === 'test' && hasFactories) {
+        server.loadFactories(modulesMap['factories']);
+      } else if (hasDefaultScenario && hasFactories) {
```

Comment:
I think we also need a `env !== 'test'` here, since we don't want these running during testing